File: testprojects/master_scala_mainone/pipelines/ScalaDoAnything/code/src/main/scala/io/prophecy/pipelines/scaladoanything/graph/Subgraph_1/Filter_2.scala
package io.prophecy.pipelines.scaladoanything.graph.Subgraph_1

import io.prophecy.libs._
import io.prophecy.pipelines.scaladoanything.config.ConfigStore._
import io.prophecy.pipelines.scaladoanything.udfs.UDFs._
import io.prophecy.pipelines.scaladoanything.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Filter_2 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.filter(lit(true))

}

File: testprojects/master_python_mainone/pipelines/PYTHON_UC/code/python_uc/graph/Subgraph_1/Filter_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from python_uc.config.ConfigStore import *
from python_uc.udfs.UDFs import *

def Filter_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter(lit(True))
